file/cache.lisp

```diff
@@ -5,24 +5,32 @@
   (:use :cl)
   (:export #:file-cache-stream
            #:file-cache-character-stream
-           #:direction
-           #:buffer
-           #:file-position*
-           #:file-length*))
+           #:allocate-new-block
+           #:read-file-block
+           #:write-file-block))
 
 (in-package :mezzano.file-system-cache)
 
 (defclass file-cache-stream (file-stream)
```

Comment:
file-cache-stream should inherit from fundamental-binary-input-stream and fundamental-binary-output-stream as it implements both, then file systems would only need to inherit from file-cache-stream, not the Gray stream classes.